docs/reference/benchmarks.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: default
+title: Benchmarks
+nav_order: 4
+permalink: /reference/benchmarks/
+parent: Reference
+---
+
+This page describes each benchmark that is part of fuzzbench.
+
+The table below was generated using benchmarks.py.
+
+| Benchmark                   | Fuzz target            | Has dictionary? | Number of seed inputs | Number of progam edges | Size (MB) |
```

Comment:
Do we need the "Fuzz target" column? It doesn't seem very important/useful.

Reply:
Over time, this will be lot of oss-fuzz benchmarks, so i think it is useful.

Reply:
Yeah was feeling conflicted about this since the info is somewhat redundant and it's taking up expensive real estate in the second column. But I think it's important to put somewhere.

Reply:
If we could turn them into links to the fuzz target source code, that'd be very useful.

Reply:
I don't know of an automated way to do this.